### PR TITLE
Move to C++20

### PR DESCRIPTION
**Pull request by ktf**

### CODE REVIEW

**Comment by ktf** [comment]
Let's see how much it breaks.

**Comment by ddobrigk** [comment]
Okay, I think we missed merging this - will merge and then if we get troubles it wil be from tomorrow's tag (Monday). Let's see....

**Review by ddobrigk — APPROVED** [review]

**Comment by ddobrigk** [comment]
Will wait until end of Sunday to merge to master, stop me if not ok!

**Pull request merged**
